templates/ts/action/src/actionApp.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  TeamsActivityHandler,
+  CardFactory,
+  TurnContext,
+  MessagingExtensionResponse,
+  MessagingExtensionAction,
+  MessagingExtensionActionResponse,
+} from "botbuilder";
+
+export class ActionApp extends TeamsActivityHandler {
+  //Action
+  public async handleTeamsMessagingExtensionSubmitAction(
+    context: TurnContext,
+    action: MessagingExtensionAction
+  ): Promise<MessagingExtensionActionResponse> {
+    switch (action.commandId) {
+      case "createCard":
+        return createCardCommand(context, action);
+      default:
+        throw new Error("NotImplemented");
+    }
+  }
+}
+
+async function createCardCommand(
```

Comment:
After discussing with @MuyangAmigo, we decide to remove "shareMessage" command and add entry point for "createCard" command to showcase me action's usage.